build/Targets/Packages.props

```diff
@@ -45,7 +45,7 @@
     <MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>14.3.25407-alpha</MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta1-61531-03</MicrosoftMetadataVisualizerVersion>
     <MicrosoftMSXMLVersion>8.0.0.0-alpha</MicrosoftMSXMLVersion>
-    <MicrosoftNetCompilersVersion>2.3.0-beta3-61821-14</MicrosoftNetCompilersVersion>
+    <MicrosoftNetCompilersVersion>2.6.0-beta1-61906-06</MicrosoftNetCompilersVersion>
```

Comment:
roslyn analyzer (https://github.com/dotnet/roslyn-analyzers/blob/master/build/Versions.props) we are using in this branch requires the version.